app/services/load_statements.rb

```diff
@@ -15,6 +15,7 @@ def run
       touched_statements = csv.map { |result| parse_result(result) }
       untouched_statements = page.statements.where.not(id: touched_statements)
       untouched_statements.update(removed_from_source: true)
+      touched_statements.map(&:save!)
```

Comment:
Should this be `touched_statements.each(&:save!)`?

Reply:
As were not using the result, yes. Good catch